Author: Roberto C. Sanchez <roberto@connexer.com>
Description: Fix lintian errors (privacy-breach-w3c-valid-html & privacy-breach-logo) by removing remote image references in doc/index.html
--- luabind.git.orig/doc/index.html
+++ luabind.git/doc/index.html
@@ -41,23 +41,6 @@
 
 <p>
 luabind is hosted by sourceforge.<br>
-    <a href="http://sourceforge.net">
-    <img src="http://sourceforge.net/sflogo.php?group_id=77060&amp;type=5"
-    style="width:210;height:62;border:0"
-    alt="SourceForge.net Logo"></a>
-
-<br>
-
-    <a href="http://validator.w3.org/check/referer">
-    <img style="border:0;width:88px;height:31px"
-    src="http://www.w3.org/Icons/valid-html401"
-    alt="Valid HTML 4.01!"></a>
-
-    <a href="http://jigsaw.w3.org/css-validator/">
-    <img style="border:0;width:88px;height:31px"
-    src="http://jigsaw.w3.org/css-validator/images/vcss"
-    alt="Valid CSS!"></a>
-
 </p>
 
 </body></html>
